test/controllers/concerns/metadata_test.rb

```diff
@@ -16,10 +16,19 @@ def authenticate
 
   test 'meta adds total and search to JSON response' do
     authenticate
+    3.times do
+      Profile.create(ref_id: "foo#{SecureRandom.uuid}", name: SecureRandom.uuid,
+                     benchmark: benchmarks(:one), hosts: [hosts(:one)],
+                     account: accounts(:test))
+    end
     profiles(:one).update(account: accounts(:test), hosts: [hosts(:one)])
-    search_query = 'name=profile1'
-    get profiles_url, params: { search: search_query }
+    search_query = 'ref_id~foo'
+    get profiles_url, params: { search: search_query, limit: 1, offset: 2 }
```

Comment:
Line 32 of this test was supposed to be testing the total count using the search param, the test passed in the past because there was only 1 profile in the db. So I'm making the db and results a bit larger so that if you remove the changes I made on the controller to compute the count based on the scope + search, the test would fail.